shopfloor_reception/__manifest__.py

```diff
@@ -9,7 +9,11 @@
     "maintainers": ["mmequignon", "JuMiSanAr"],
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["shopfloor"],
+    "depends": [
+        "shopfloor",
+        # TODO: Used to parse return barcodes coming from carriers
```

Comment:
Why TODO?

Reply:
Because the manifest is written like that for shopfloor, and I moved this to `shopfloor_reception` to address @simahawk 's comment.